### DISCUSSION TITLE
Subcription Id is not working

### DISCUSSION THREAD

**Discussion by nnedelea** [potmat/homebridge-google-nest-sdm#55] [Q&A]
Hello,

The subscription id gives this error:

ERROR: (gcloud.pubsub.subscriptions.create) unrecognized arguments:
  pubsub
  subscriptions
  create
  subscription-id1
  Created
  subscription
  [projects/gcp-project-name/subscriptions/subscription-id1].
  To search the help text of gcloud commands, run:
  gcloud help -- SEARCH_TERMS

Please contact to resolve this issue.

**Comment by potmat** [comment]
Hi @nnedelea, I'm not able to provide support for going through the getting started guide. The Nest plugin for Home Assistant uses much the same process (don't forget the ["ONE IMPORTANT DIFFERENCE"](https://github.com/potmat/homebridge-google-nest-sdm#where-do-the-config-values-come-from) section above). It has an [illustrated guide](https://www.home-assistant.io/integrations/nest/) that you may find helpful. You can also try reaching out to others on [Discord](https://discord.gg/kqNCe2D) or [Reddit](https://www.reddit.com/r/homebridge/), some people there may be able to help.

I've converted this issue to a discussion, hopefully other users of the plugin can help you out.
